docs/context/ref/Action.md

```diff
@@ -0,0 +1,65 @@
+---
+id: Action
+sidebar_label: Action
+title: Action
+hide_title: true
+---
+# `Action`
+
+A representation of an FDC3 Action (specified via a Context or Context & Intent) that can be inserted inside another object, 
+for example a chat message.
+
+The action may be completed by calling `fdc3.raiseIntent()` with the specified Intent and Context, or, if only a context is specified, by calling `fdc3.raiseIntentForContext()` (which the Desktop Agent will resolve by presenting the user with a list of available Intents for the Context).
+
+## Type
+
+`fdc3.action`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/action.schema.json
+
+## Details
+
+| Property          | Type                                      | Required | Example Value           |
+|-------------------|-------------------------------------------|----------|-------------------------|
+| `type`            | string                                    | Yes      | `'fdc3.action'`  |
+| `title`           | string                                    | Yes      | `'Click to view Chart'` |
+| `context`         | string                                    | Yes      | See Below               |
+| `intent`          | string                                    | No       | `'ViewChart'`           |
+
+## Example
+
+```js
+const message = {
```

Comment:
Example shouldn't be based on message unless you explain it as a use-case first ;-)

Reply:
Also doesn't match spec of proposed message format